src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -99,9 +101,9 @@ async def get_output_volts(self):
         :rtype: float
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_OUTPUTVOLTS, [MGMSG.PZ_GET_OUTPUTVOLTS], 1
+            MGMSG.PZ_REQ_OUTPUTVOLTS, [MGMSG.PZ_GET_OUTPUTVOLTS], _CHANNEL
         )
-        return st.unpack("<H", get_msg.data[2:])[0] * self.voltage_limit / 32767
+        return st.unpack("<H", get_msg.data[2:])[0] * self.voltage_limit / 65476
```

Comment:
Why does this number need to change? Is it different between K-Cubes and T-Cubes? If the two devices require different parameters here, this method should be overridden by `Kpz` so that this function doesn't now have a bug on TPZ001s.